File: src/entitie/Enfant.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package entitie;

/**
 *
 * @author Hejer Ghariani
 */
public class Enfant {
    private int id_enfant;
    private String nom;
    private String prenom;
    private int age;
    private String photo;
    private int id_parent;
    private String vaccin;
    private String alergie;
    private Paarent parent;
    private int idjardin;
    private String email;

    public Enfant() {
    }
    
    
    

    public Enfant(int id_enfant, String nom, String prenom, int age, String photo, int id_parent, String vaccin, String alergie) {
        this.id_enfant = id_enfant;
        this.nom = nom;
        this.prenom = prenom;
        this.age = age;
        this.photo = photo;
        this.id_parent = id_parent;
        this.vaccin = vaccin;
        this.alergie = alergie;
    }
    public Enfant(int id_enfant, String nom, String prenom, int age, String photo, int id_parent, String vaccin, String alergie,String email) {
        this.id_enfant = id_enfant;
        this.nom = nom;
        this.prenom = prenom;
        this.age = age;
        this.photo = photo;
        this.id_parent = id_parent;
        this.vaccin = vaccin;
        this.alergie = alergie;
        this.email=email;
    }
    
    

    public Enfant(int id_enfant, String nom, String prenom, int age, String photo, int id_parent, String vaccin, String alergie, Paarent parent, int idjardin,String email) {
        this.id_enfant = id_enfant;
        this.nom = nom;
        this.prenom = prenom;
        this.age = age;
        this.photo = photo;
        this.id_parent = id_parent;
        this.vaccin = vaccin;
        this.alergie = alergie;
        this.parent = parent;
        this.idjardin = idjardin;
        
    }

    public Enfant(int id_enfant, String nom, String prenom, int age, String photo, int id_parent, String vaccin, String alergie, int idjardin ,String email) {
        this.id_enfant = id_enfant;
        this.nom = nom;
        this.prenom = prenom;
        this.age = age;
        this.photo = photo;
        this.id_parent = id_parent;
        this.vaccin = vaccin;
        this.alergie = alergie;
        this.idjardin = idjardin;
        this.email=email;
    }

    public int getIdjardin() {
        return idjardin;
    }

    public void setIdjardin(int idjardin) {
        this.idjardin = idjardin;
    }
    
    

    public Enfant(String nom, String prenom, int age, String photo, int id_parent, String vaccin, String alergie, int  idjardin) {
        this.nom = nom;
        this.prenom = prenom;
        this.age = age;
        this.photo = photo;
        this.id_parent = id_parent;
        this.vaccin = vaccin;
        this.alergie = alergie;
        this.idjardin = idjardin;
    }

 
    

    public Enfant(String nom, String prenom, int age, String photo, int id_parent, String vaccin, String alergie) {
        this.nom = nom;
        this.prenom = prenom;
        this.age = age;
        this.photo = photo;
        this.id_parent = id_parent;
        this.vaccin = vaccin;
        this.alergie = alergie;
    }
    

    public int getId_enfant() {
        return id_enfant;
    }

    public String getNom() {
        return nom;
    }

    public String getPrenom() {
        return prenom;
    }

    public int getAge() {
        return age;
    }

    public String getPhoto() {
        return photo;
    }

    public int getId_parent() {
        return id_parent;
    }

    public String getVaccin() {
        return vaccin;
    }

    public String getAlergie() {
        return alergie;
    }

    public void setId_enfant(int id_enfant) {
        this.id_enfant = id_enfant;
    }

    public void setNom(String nom) {
        this.nom = nom;
    }

    public void setPrenom(String prenom) {
        this.prenom = prenom;
    }

    public void setAge(int age) {
        this.age = age;
    }

    public void setPhoto(String photo) {
        this.photo = photo;
    }

    public void setId_parent(int id_parent) {
        this.id_parent = id_parent;
    }

    public void setVaccin(String vaccin) {
        this.vaccin = vaccin;
    }

    public void setAlergie(String alergie) {
        this.alergie = alergie;
    }

    @Override
    public String toString() {
        return "Enfant{" + "id_enfant=" + id_enfant + ", nom=" + nom + ", prenom=" + prenom + ", age=" + age + ", photo=" + photo + ", id_parent=" + id_parent + ", vaccin=" + vaccin + ", alergie=" + alergie + ", idjardin=" + idjardin + ", email=" + email + '}';
    }

    
    
    
    
    
}

File: PIDEVFrogs/src/MobileBundle/Entity/Enfant.php
<?php

namespace MobileBundle\Entity;

use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\Validator\Constraints as Assert;

/**
 * Enfant
 *
 * @ORM\Table(name="enfant", indexes={@ORM\Index(name="FK_34B70CA250EAE44", columns={"id_utilisateur"})})
 * @ORM\Entity(repositoryClass="JardinBundle\Repository\EnfantRepository")
 */
class Enfant
{
    /**
     * @var integer
     *
     * @ORM\Column(name="id_enfant", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idEnfant;

    /**
     * @var string
     *
     * @ORM\Column(name="nom", type="string", length=11, nullable=false)
     * @Assert\Length(
     *     min = 5,
     *     max = 20,
     *     minMessage = "Your first name must be at least {{ limit }} characters long",
     *     maxMessage = "Your first name cannot be longer than {{ limit }} characters"
     * )
     */
    private $nom;

    /**
     * @var string
     *
     * @ORM\Column(name="prenom", type="string", length=11, nullable=false)* @Assert\Length(
     *     min = 5,
     *     max = 20,
     *     minMessage = "Your first name must be at least {{ limit }} characters long",
     *     maxMessage = "Your first name cannot be longer than {{ limit }} characters"
     * )
     */
    private $prenom;

    /**
     * @var integer
     *
     * @ORM\Column(name="age", type="integer", nullable=false)
     */
    private $age;

    /**
     * @var string
     *
     * @ORM\Column(name="photo", type="text", length=65535, nullable=false)
     *@Assert\GreaterThan(0)
     */
    private $photo;

    /**
     * @var boolean
     *
     * @ORM\Column(name="vaccin", type="boolean", nullable=false)
     */
    private $vaccin=false;

    /**
     * @var boolean
     *
     * @ORM\Column(name="alergie", type="boolean", nullable=false)
     */
    private $alergie=false;



    /**
     * @var \User
     *
     * @ORM\ManyToOne(targetEntity="UserBundle\Entity\User")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="id_utilisateur", referencedColumnName="id")
     * })
     */
    private $idUtilisateur;



    /**
     * Get idEnfant
     *
     * @return integer
     */
    public function getIdEnfant()
    {
        return $this->idEnfant;
    }

    /**
     * Set nom
     *
     * @param string $nom
     *
     * @return Enfant
     */
    public function setNom($nom)
    {
        $this->nom = $nom;

        return $this;
    }

    /**
     * Get nom
     *
     * @return string
     */
    public function getNom()
    {
        return $this->nom;
    }

    /**
     * Set prenom
     *
     * @param string $prenom
     *
     * @return Enfant
     */
    public function setPrenom($prenom)
    {
        $this->prenom = $prenom;

        return $this;
    }

    /**
     * Get prenom
     *
     * @return string
     */
    public function getPrenom()
    {
        return $this->prenom;
    }

    /**
     * Set age
     *
     * @param integer $age
     *
     * @return Enfant
     */
    public function setAge($age)
    {
        $this->age = $age;

        return $this;
    }

    /**
     * Get age
     *
     * @return integer
     */
    public function getAge()
    {
        return $this->age;
    }

    /**
     * Set photo
     *
     * @param string $photo
     *
     * @return Enfant
     */
    public function setPhoto($photo)
    {
        $this->photo = $photo;

        return $this;
    }

    /**
     * Get photo
     *
     * @return string
     */
    public function getPhoto()
    {
        return $this->photo;
    }

    /**
     * Set vaccin
     *
     * @param boolean $vaccin
     *
     * @return Enfant
     */
    public function setVaccin($vaccin)
    {
        $this->vaccin = $vaccin;

        return $this;
    }

    /**
     * Get vaccin
     *
     * @return boolean
     */
    public function getVaccin()
    {
        return $this->vaccin;
    }

    /**
     * Set alergie
     *
     * @param boolean $alergie
     *
     * @return Enfant
     */
    public function setAlergie($alergie)
    {
        $this->alergie = $alergie;

        return $this;
    }

    /**
     * Get alergie
     *
     * @return boolean
     */
    public function getAlergie()
    {
        return $this->alergie;
    }


    /**
     * Set idUtilisateur
     *
     * @param \User\Bundle\Entity\User $idUtilisateur
     *
     * @return Enfant
     */
    public function setIdUtilisateur(\UserBundle\Entity\User $idUtilisateur )
    {
        $this->idUtilisateur = $idUtilisateur;

        return $this;
    }


    /**
     * Get idUtilisateur
     *
     * @return \JardinBundle\Entity\User
     */
    public function getIdUtilisateur()
    {
        return $this->idUtilisateur;
    }

}
